feature-libs/my-account/organization/components/shared/organization-detail/toggle-status-action/toggle-status.component.ts

```diff
@@ -120,14 +131,6 @@ export class ToggleStatusComponent<T extends BaseItem> implements OnDestroy {
     this.itemActiveState = { ...item };
```

Comment:
The `itemActiveState` is no longer needed, as it was used before to verify if the item was activated. we should drop this line as well as the `itemActiveState` property.

Reply:
cleaned